vendor/vendor-apple/base/src/main/kotlin/com/malinskiy/marathon/apple/xctestrun/TestRootFactory.kt

```diff
@@ -209,6 +211,7 @@ class TestRootFactory(
                     arrayOf(
                         TestTarget.withArtifactReinstall(
                             name = bundle.testBundleId,
+                            environmentVariables = xctestrunEnv.appEnvs,
```

Comment:
Not sure if empty works here, maybe we should omit this in case of empty envvars